CHANGES.rst

```diff
@@ -5,6 +5,11 @@
 
    - feature message
 
+1.5.1
```

Comment:
This is not a bug fix, it's a new feature, so should be 1.6.0

Reply:
Fixed.